src/resmom/mom_main.c

```diff
@@ -3429,8 +3432,20 @@ add_static(char *str, char *file, int linenum)
 
 	str = TOKCPY(str, name);/* resource name */
 	str = skipwhite(str);	/* resource value */
-	if (*str == '!')	/* shell escape command */
+	if (*str == '!') {	/* shell escape command */
+		int err;
 		rmnl(str);
+#ifdef  WIN32
+		err = tmp_file_sec(&str[1], 0, 1, WRITES_MASK, 1);
+#else
+		err = tmp_file_sec(&str[1], 0, 1, S_IWGRP|S_IWOTH, 1);
+#endif
+		if (err != 0) {
+			snprintf(log_buffer, sizeof(log_buffer), "error: %s file has a non-secure file access, errno: %d", &str[1], err);
+			log_event(PBSEVENT_DEBUG, PBS_EVENTCLASS_SERVER, LOG_DEBUG, __func__, log_buffer);
```

Comment:
There's a 'PBSEVENT_SECURITY' event type, should we be logging at that? I also feel that LOG_DEBUG might be too low, shouldn't it be LOG_ERR?